entrypoint.sh

```diff
@@ -17,7 +17,7 @@ case "${GITHUB_EVENT_NAME}" in
 push|create|pull_request|workflow_dispatch)
   git-setup
   git fetch --all
-  git push -f --all target
+  git push -f ${INPUT_SKIPCI:+-o skip.ci} --all target
```

Comment:
```suggestion
  git push -f ${INPUT_SKIPCI:+-o ci.skip} --all target
```
I believe, `skip.ci` was always supported (at least back in the day on Travis), but only for typo-friendliness.
https://docs.gitlab.com/ee/user/project/push_options.html